File: chap15_muscle/exercises/assay2.m
clear all
close all
clc

syms A g2
% f1 = 65;
% g1 = 15;
% g2 = 10;

out1 = g2/2 + A/24;
out2 = g2/sqrt(2) - g2*g2/(2*A);

dd = out1-out2;
root = solve(diff(dd,g2)==0,g2)
simplify(subs(dd,g2,root))

File: chap15_muscle/exercises/assay2.py
from sympy import *

A,g2 = symbols("A g2")

vl = g2/2 + A/24;
vs = g2/sqrt(2) - g2*g2/(2*A);

dd = vl - vs
g2m = solve(diff(dd,g2),g2)
print(simplify(dd.subs(g2,g2m[0])))
